TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/Index.cshtml

```diff
@@ -7,32 +7,61 @@
     ViewBag.PersonId = Model.PersonId;
     ViewBag.Search = Model.Search;
     ViewBag.SortBy = Model.SortBy;
-    ViewBag.PageNumber = Model.PageNumber;    
+    ViewBag.PageNumber = Model.PageNumber;
     ViewBag.Title = Model.Person!.Name;
 }
 
+@if (Model.Person.HasAlerts)
+{
+    <govuk-notification-banner>
+        <p class="govuk-notification-banner__heading">
+            Alert on record.
+            <a class="govuk-notification-banner__link" href="@LinkGenerator.PersonAlerts(Model.PersonId, Model.Search, Model.SortBy, Model.PageNumber)">View alert</a>.
+        </p>
+    </govuk-notification-banner>
+
+}
+
 <govuk-summary-card>
     <govuk-summary-card-title>Personal Details</govuk-summary-card-title>
     <govuk-summary-list data-testid="personal-details">
         <govuk-summary-list-row>
             <govuk-summary-list-row-key>Name</govuk-summary-list-row-key>
             <govuk-summary-list-row-value data-testid="personal-details-name">@Model.Person.FullName</govuk-summary-list-row-value>
         </govuk-summary-list-row>
+        @if (Model.Person.PreviousNames.Length > 0)
+        {
+            @for (int i = 0; i < Model.Person.PreviousNames.Length; i++)
+            {
+                <govuk-summary-list-row>
```

Comment:
I know the designs have each previous name in their own row but I think we should have a single row here with a `<ul>` with the list of names.

Reply:
done